doc/src/site/sphinx/migration_guide.rst

```diff
@@ -47,6 +47,40 @@ From 3.30 to 3.32
   than 0, validation metrics if validation frame has been specified ( splitRatio was set and lower than 1 ) and nfolds was 0
   and training metrics otherwise ( splitRatio is 1 and nfolds is 0).
 
+- The whole trait ``ModelSerializationSupport`` in Scala is removed. The MOJO is a first class citizen in Sparkling Water and
+  most code works with our Spark MOJO wrapper. Please use the following approaches to migrate from previous methods
+  in the model serialization support:
+
+    To crete Spark MOJO wrapper in Sparkling Water, you can load it from H2O-3 as:
+
+    .. code-block:: scala
+
+        val mojoModel = H2OMOJOModel.createFromMojo(path)
```

Comment:
Since there are also specific mojo model types (`H2OTreeBasedSupervisedMOJOModel`), I would add link to this page: http://docs.h2o.ai/sparkling-water/2.2/latest-stable/doc/deployment/load_mojo.html